docs/user-guide/address-security-requirements.md

```diff
@@ -0,0 +1,73 @@
+| ZSS (required for API ML certificate and identity mapping) | APPL                                    | `OMVSAPPL`                  | READ   | Allows Zowe to run code on behalf of the API requester's TSO user ID. This permission is also required from a requester's TSO user. You can skip this requirement when the resource `OMVSAPPL` in the `APPL` class is not defined. For more information, see [Security Environment Switching](./configure-zos-system.md#configure-security-environment-switching). | Run the commands to permit the Zowe main server to run the code on behalf of the user. |                                                                                                                                                                                                                                                                                                                                                                        |
+| Core                                                       | FACILITY                                                    | `BPX.JOBNAME`               | READ   | To allow z/OS address spaces for unix processes to be renamed for [ease of identification](./configure-zos-system.md#configure-address-space-job-naming). | Run the commands to permit the Zowe main server to set the job name.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Core?                                                      | FACILITY                                                   | `IRR.RADMIN.LISTUSER`       | READ   | To allow Zowe to obtain information about OMVS segment of the user profile using `LISTUSER` TSO command.   | Run the commands to permit Zowe ... |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
I don't know what this is good for. If no one else clarifies then just remove '?'.